ui/app/AppLayouts/Browser/BrowserLayout.qml

```diff
@@ -122,16 +122,656 @@ Item {
         registeredObjects: [provider]
     }
 
+    property QtObject defaultProfile: WebEngineProfile {
+        storageName: "Profile"
+        offTheRecord: false
+        useForGlobalCertificateVerification: true
+        userScripts: [
+            WebEngineScript {
+                injectionPoint: WebEngineScript.DocumentCreation
+                sourceUrl:  Qt.resolvedUrl("provider.js")
+                worldId: WebEngineScript.MainWorld // TODO: check https://doc.qt.io/qt-5/qml-qtwebengine-webenginescript.html#worldId-prop 
+            }
+        ]
+    }
+
+    property QtObject otrProfile: WebEngineProfile {
+        offTheRecord: true
+        persistentCookiesPolicy:  WebEngineProfile.NoPersistentCookies
+        userScripts: [
+            WebEngineScript {
+                injectionPoint: WebEngineScript.DocumentCreation
+                sourceUrl:  Qt.resolvedUrl("provider.js")
+                worldId: WebEngineScript.MainWorld // TODO: check https://doc.qt.io/qt-5/qml-qtwebengine-webenginescript.html#worldId-prop 
+            }
+        ]
+    }
+
+    onCurrentWebViewChanged: {
+        findBar.reset();
+    }
+
+    Settings {
+        id : appSettings
+        property alias autoLoadImages: loadImages.checked
+        property alias javaScriptEnabled: javaScriptEnabled.checked
+        property alias errorPageEnabled: errorPageEnabled.checked
+        property alias pluginsEnabled: pluginsEnabled.checked
+        property alias autoLoadIconsForPage: autoLoadIconsForPage.checked
+        property alias touchIconsEnabled: touchIconsEnabled.checked
+        property alias webRTCPublicInterfacesOnly : webRTCPublicInterfacesOnly.checked
+        property alias devToolsEnabled: devToolsEnabled.checked
+        property alias pdfViewerEnabled: pdfViewerEnabled.checked
+    }
+
+    Action {
+        shortcut: "Ctrl+D"
```

Comment:
Oh I didn't know we could do that. We could do that for some chat shortcuts as well maybe (unrelated to your PR)